dftimewolf/lib/containers/manager.py

```diff
@@ -0,0 +1,137 @@
+"""A ContainerManager class."""
+
+
+import dataclasses
+import threading
+import typing
+from typing import cast, Sequence, Type
```

Comment:
maybe just directly import `Any` so you are not importing the whole typing package

Reply:
Done.